frontend/src/components/Layout.tsx

```diff
@@ -5,17 +5,25 @@ import { useTranslation } from 'next-i18next'
 import Footer from './Footer'
 import Header from './Header'
 
+type BreadcrumbItemType = {
```

Comment:
This should be imported from the header component and then re-exported here.

Reply:
After some additional thought.. ignore this comment. I think that `BreadcrumbItemType` should be exported from `Breadcrumb.tsx` and then used wherever necessary.